api/services/rd-professional.ts

```diff
@@ -0,0 +1,14 @@
+import * as log4js from 'log4js'
+import config from '../lib/config'
+import { http } from '../lib/http'
+
+const logger = log4js.getLogger('rd-professional')
```

Comment:
This needs to be changed to log4jui and the level line can be removed

Reply:
Logging removed as not required.